src/libs/events.ts

```diff
@@ -224,13 +226,16 @@ export const extractWorkspaceDetails = (workspaceObject: EventWorkspaceAttribute
   // For other types of input, whether the workspace has protected data is unknown.
   const hasProtectedData =
     'policies' in workspaceObject ? containsProtectedDataPolicy(workspaceObject.policies) : undefined;
+  const hasPhiTracking =
+    'policies' in workspaceObject ? containsPhiTrackingPolicy(workspaceObject.policies) : undefined;
 
```

Comment:
Example of event data on a workspace event:

<img width="789" alt="image" src="https://github.com/DataBiosphere/terra-ui/assets/484484/e8d8baf0-f4ae-453f-aae7-c74d1ca51a34">